js/ace.js

```diff
@@ -0,0 +1,1046 @@
+            timestamp = this.parse8601 (orderTime);
+            const orderSide = this.safeNumber (order, 'buyOrSell');
+            side = (orderSide === 1) ? 'buy' : 'sell';
+            amount = this.safeString (order, 'num');
+            price = this.safeString (order, 'price');
+            const quoteId = this.safeString (order, 'quoteCurrencyName');
+            const baseId = this.safeString (order, 'baseCurrencyName');
+            if (quoteId !== undefined && baseId !== undefined) {
+                symbol = baseId + '/' + quoteId;
+            }
+            const orderType = this.safeNumber (order, 'type');
+            if (orderType !== undefined) {
+                type = (orderType === 1) ? 'limit' : 'market';
+            }
+            filled = this.safeString (order, 'tradeNum');
+            remaining = this.safeString (order, 'remainNum');
+            status = this.parseOrderStatus (this.safeString (order, 'status'));
+            average = this.safeString (order, 'averagePrice');
+        }
+        return this.safeOrder ({
+            'id': id,
+            'clientOrderId': undefined,
+            'timestamp': timestamp,
+            'datetime': orderTime,
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': undefined,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'amount': amount,
+            'cost': undefined,
+            'average': average,
+            'filled': filled,
+            'remaining': remaining,
+            'status': status,
+            'fee': undefined,
+            'trades': undefined,
+            'info': undefined,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        /**
+         * @method
+         * @name ace#createOrder
+         * @description create a trade order
+         * @see https://github.com/ace-exchange/ace-official-api-docs/blob/master/api_v2.md#open-api---new-order
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market' or 'limit'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const orderType = type.toUpperCase ();
+        const orderSide = side.toUpperCase ();
+        const request = {
```

Comment:
`createOrder` is not workign because precisions are not correct, you probably need to parse them with `parsePrecision` otherwise it says the minimum amount is a huge value 

```
CCXT v2.6.47
ace.createOrder (LTC/USDT, market, buy, 0.1)
ArgumentsRequired ace amount of LTC/USDT must be greater than minimum amount precision of 2
---------------------------------------------------
[ArgumentsRequired] ace amount of LTC/USDT must be greater than minimum amount precision of 2

    at amountToPrecision          js/base/Exchange.js:2645              throw new ArgumentsRequired (this.id + ' amount of ' + market['symbol'] + ' mus…
    at createOrder                js/ace.js:599                         'num': this.amountToPrecision (symbol, amount),                                 
    at processTicksAndRejections  node:internal/process/task_queues:96                                                                                  
    at async run                  examples/js/cli.js:293                const result = await exchange[methodName] (... args)                            

ace amount of LTC/USDT must be greater than minimum amount precision of 2
```